examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala

```diff
@@ -97,22 +95,22 @@ object DFSReadWriteTest {
   def main(args: Array[String]): Unit = {
     parseArgs(args)
 
-    println("Performing local word count")
+    println(s"Performing local word count")
```

Comment:
There is no interpolation here; revert changes like this.

Reply:
@srowen Thanks for review, I did addressed changes. Please review